python-vcversioner: new package

jefferyto opened the pull request:
From the README:

Elevator pitch: you can write a setup.py with no version information
specified, and vcversioner will find a recent, properly-formatted VCS
tag and extract a version from it.

Signed-off-by: Jeffery To <jeffery.to@gmail.com>

yousong commented:
Hmm, just being curious, I guess this package is mainly for dev purposes.  Is it really necessary for OpenWrt?

yousong commented:
Oh, I now see that it is to be required by `python-txsocksx`.  But can be just patch `python-txsocksx` `setup.py` with explicit version info to make its dependency list more slim?

jefferyto commented:
I'll take a look at patching it.

jefferyto commented:
I've updated python-txsocksx to use the `find_version()` function from (python-)pip and removed the dependency on vcversioner. If the version number was hard-coded into a patch, I'm sure at some point I'll forget to update it.

jefferyto commented:
Closing as this package isn't necessary at the moment.
